modules/oc-mirror-creating-image-set-config.adoc

```diff
@@ -0,0 +1,46 @@
+// Module included in the following assemblies:
+//
+// * installing/disconnected_install/installing-mirroring-disconnected.adoc
+
+:_content-type: PROCEDURE
+[id="oc-mirror-creating-image-set-config_{context}"]
+= Creating the image set configuration
+
+Before you can use the oc-mirror plug-in to mirror image sets, you must create an image set configuration file. This image set configuration file defines which {product-title} releases, Operators, and other images to mirror, along with other configuration settings for the oc-mirror plug-in.
+
+You must specify a storage backend in the image set configuration file. This storage backend can be a local directory or a docker v2 registry. The oc-mirror plug-in stores metadata in this storage backend during image set creation.
+
+[IMPORTANT]
+====
+Do not delete or modify the metadata that is generated by the oc-mirror plug-in executions. You must use the same storage backend on subsequent executions of oc-mirror to the same mirror registry.
+====
+
+.Procedure
+
+. Create an `ImageSetConfiguration` resource that specifies the necessary configuration details:
++
+.Example `ImageSetConfiguration` file
+[source,yaml]
+----
+apiVersion: mirror.openshift.io/v1alpha1
+kind: ImageSetConfiguration
+archiveSize: 4 <1>
+mirror:
+ ocp:
+   channels:
+     - name: stable-4.9 <2>
+ operators:
+   - catalog: registry.redhat.io/redhat/redhat-operator-index:v4.9 <3>
+storageConfig: <4>
+ registry:
+   imageURL: example.com/example/oc-mirror <5>
+----
+<1> The maximum size, in GiB, of each file within the image set.
+<2> The channel to retrieve the {product-title} images from.
+<3> The Operator catalog to retrieve the {product-title} images from.
+<4> The back-end location to save the image set metadata to. This location can be a registry or local directory. It is required to specify a `storageConfig`.
+<5> The registry URL for the storage backend.
++
+This example will pull images from the `stable-4.9` channel for the `registry.redhat.io/redhat/redhat-operator-index:v4.9` operator catalog and saves the image set metadata to the `example.com/example/oc-mirror` registry.
+
+. Save the file as `imageset-config.yaml`. This file is required by the `oc mirror` command when mirroring content.
```

Comment:
+1 for it being a bit confusing to document and read. If I had my preference, it would just be an `oc mirror` command that was already part of the `oc` binary. But I'll bring it up to the team for discussion prior to GA, just in case they want to consider any changes.